### ISSUE TITLE
trying to get in touch

### ISSUE THREAD

**Issue by Aulteran** [ault/trial1#1]
hey man, sorry - I didn't know any other way to send you a message apart from making a repository issue. I was wondering i can take your github username? i noticed you're inactive on github so i thought i could take the username ault. not sure if you'll even see this message - hopefully you have notifications enabled or github sends you an email about this.

pls let me know if you're willing man :\ you can comment on this issue or respond to me or you can email me at ault@andrims.com

even if you choose not to give it to me could you reach out and let me know you wont give it? thanks man :]

**Comment by ault** [comment]
Unfortunately, github is used for a lot of our company work... I'm not sure what affect changing my handle might cause.

On the plus side, there is some reason to believe that I may not need it after Q2 2024. If you check back with me some time around May next year I may be in a position to pass it along.

BTW - is Ault your surname?

**Comment by Aulteran** [comment]
Yes, Ault is my name. thanks for responding, I don't mind waiting for May 2024 haha :) i appreciate the gesture.
